package.json

```diff
@@ -38,63 +45,63 @@
-      "files": ["bin/osx/**/*"]
+      "target": [
+        "dmg",
+        "zip"
+      ],
+      "files": [
+        "bin/osx/**/*"
+      ]
     },
     "linux": {
       "artifactName": "${productName}-${version}-${arch}-linux.${ext}",
       "maintainer": "bryphe@outlook.com",
-      "target": ["tar.gz", "deb", "rpm"]
+      "target": [
+        "tar.gz",
+        "deb",
+        "rpm"
+      ]
     },
     "win": {
-      "target": ["zip", "dir"],
-      "files": ["bin/x86/**/*"]
+      "target": [
+        "zip",
+        "dir"
+      ],
+      "files": [
+        "bin/x86/**/*"
+      ]
     }
   },
   "scripts": {
-    "build":
-      "npm run build:browser && npm run build:main && npm run build:plugins",
+    "build": "npm run build:browser && npm run build:main && npm run build:plugins",
     "build-debug": "npm run build:browser-debug && npm run build:plugins",
     "build:browser": "webpack --config browser/webpack.production.config.js",
     "build:browser-debug": "webpack --config browser/webpack.debug.config.js",
     "build:main": "cd main && tsc -p tsconfig.json",
     "build:plugins": "npm run build:plugin:oni-plugin-typescript",
-    "build:plugin:oni-plugin-typescript":
-      "cd vim/core/oni-plugin-typescript && npm run build",
+    "build:plugin:oni-plugin-typescript": "cd vim/core/oni-plugin-typescript && npm run build",
     "build:test": "cd test && tsc -p tsconfig.json",
     "pack": "build --publish never",
     "copy-icons": "node build/CopyIcons.js",
     "dist:win": "build --arch ia32 --publish never",
-    "pack:win":
-      "node build/BuildSetupTemplate.js && innosetup-compiler dist/setup.iss --verbose --O=dist",
+    "pack:win": "node build/BuildSetupTemplate.js && innosetup-compiler dist/setup.iss --verbose --O=dist",
     "test": "npm run test:unit && npm run test:integration",
-    "test:integration":
-      "npm run build:test && mocha -t 30000 lib_test/test/CiTests.js",
-    "test:stability":
-      "npm run build:test && mocha -t 600000 --recursive lib_test/test/stability",
+    "test:integration": "npm run build:test && mocha -t 30000 lib_test/test/CiTests.js",
+    "test:stability": "npm run build:test && mocha -t 600000 --recursive lib_test/test/stability",
     "test:unit": "npm run test:unit:browser",
-    "test:unit:browser":
-      "cd browser && tsc -p tsconfig.json && mocha --recursive ../lib_test/browser/test",
-    "fix-lint":
-      "npm run fix-lint:browser && npm run fix-lint:main && npm run fix-lint:test",
-    "fix-lint:browser":
-      "tslint --fix --project browser/tsconfig.json --config tslint.json && tslint --fix --project vim/core/oni-plugin-typescript/tsconfig.json --config tslint.json",
-    "fix-lint:main":
-      "tslint --fix --project main/tsconfig.json --config tslint.json",
-    "fix-lint:test":
-      "tslint --fix --project test/tsconfig.json --config tslint.json",
+    "test:unit:browser": "cd browser && tsc -p tsconfig.json && mocha --recursive ../lib_test/browser/test",
```

Comment:
Looks like `yarn` autoformatted all of these after running `yarn upgrade oni-api@0.0.6`, but there is no functional change